sklearn/preprocessing/_encoders.py

```diff
@@ -1163,6 +1182,29 @@ class OrdinalEncoder(OneToOneFeatureMixin, _BaseEncoder):
 
         .. versionadded:: 1.1
 
+    min_frequency : int or float, default=None
+        Specifies the minimum frequency below which a category will be
+        considered infrequent.
+
+        - If `int`, categories with a smaller cardinality will be considered
+          infrequent.
+
+        - If `float`, categories with a smaller cardinality than
+          `min_frequency * n_samples`  will be considered infrequent.
+
+        .. versionadded:: 1.3
+            Read more in the :ref:`User Guide <encoder_infrequent_categories>`.
+
+    max_categories : int, default=None
+        Specifies an upper limit to the number of output categories for each input
+        feature when considering infrequent categories. If there are infrequent
+        categories, `max_categories` includes the category representing the
+        infrequent categories along with the frequent categories. If `None`,
+        there is no limit to the number of output features.
+
```

Comment:
Please also explain that the `max_categories` parameter does not take into account the encoding of missing and unknown value and that depending on the values of the `handle_unknown`, `unknown_value` and `encoded_missing_value` parameters, it's possible that categorical data can be encoded with up to `max_categories + 2` unique integer codes.